src/grammar/parser-error.ts

```diff
@@ -74,11 +74,13 @@ export const syntaxErrorSchema = <T, U>(
             return handle(sub`Expression is not callable`);
         },
         noBouncedWithoutArg: () => {
-            return handle(sub`bounced() cannot be used as fallback`);
+            return handle(
+                sub`bounced() receiver should accept one of Message, bounced<Message> or Slice`,
```

Comment:
```suggestion
                sub`bounced() receiver should accept a Message, bounced<Message> or Slice`,
```

Not sure if it's better, but it's more concise

Reply:
Fixed